### PR TITLE
[zh] Translate tasks/configure-pod-container/security-context.md

### PR DESCRIPTION
**Pull request by tengqm**
closes: #22839

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+* [Linux Capabilities](https://linux-audit.com/linux-capabilities-hardening-linux-binaries-by-removing-setuid/): 
+  Give a process some privileges, but not all the privileges of the root user.
+-->
+安全上下文（Security Context）定义 Pod 或 Container 的特权与访问控制设置。
+安全上下文包括但不限于：
+
+* 自主访问控制（Discretionary Access Control）：基于
+  [用户 ID（UID）和组 ID（GID）](https://wiki.archlinux.org/index.php/users_and_groups).
+  来判定对对象（例如文件）的访问权限。
+* [安全性增强的 Linux（SELinux）](https://en.wikipedia.org/wiki/Security-Enhanced_Linux)：
```
Comment (sftim) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:35]: Is https://zh.wikipedia.org/wiki/%E5%AE%89%E5%85%A8%E5%A2%9E%E5%BC%BA%E5%BC%8FLinux sufficiently good to link to? If not, it makes sense to link to https://en.wikipedia.org/wiki/Security-Enhanced_Linux.

File: content/zh/docs/tasks/configure-pod-container/security-context.md [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
+
+<!--
+This field has no effect on ephemeral volume types such as
+[`secret`](/docs/concepts/storage/volumes/#secret),
+[`configMap`](/docs/concepts/storage/volumes/#configmap),
+and [`emptydir`](/docs/concepts/storage/volumes/#emptydir).
+-->
+{{< note >}}
+此字段对于[`Secret`](/zh/docs/concepts/storage/volumes/#secret)、
```
Comment (sftim) [inline content/zh/docs/tasks/configure-pod-container/security-context.md:303]:
```suggestion
此字段对于[`secret`](/zh/docs/concepts/storage/volumes/#secret)、
```

**Review by sftim — COMMENTED** [review]
I spotted a couple of minor things that I think you could tweak if you want to.

File: content/zh/docs/tasks/configure-pod-container/security-context.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/security-context.md:35]: Good point. Will verify if the Chinese version of wikipedia page is available.

File: content/zh/docs/tasks/configure-pod-container/security-context.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/configure-pod-container/security-context.md:303]: correct, the volume type is `secret` rather than `Secret`.

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Review by SataQiu — APPROVED** [review]
/lgtm
/approve

**Pull request merged**